README.md

```diff
@@ -9,15 +9,18 @@ The SCP Ingest Pipeline is an ETL pipeline for single-cell RNA-seq data.
 
 # Prerequisites
 
-- Python 3.7+
+- Python 3.10
 - Google Cloud Platform project
 - Suitable service account (SA) and MongoDB VM in GCP. SA needs roles "Editor", "Genomics Pipelines Runner", and "Storage Object Admin". Broad Institute engineers: see instructions [here](https://github.com/broadinstitute/single_cell_portal_configs/tree/master/terraform-mongodb).
 - SAMtools, if using `ingest/make_toy_data.py`
 - Tabix, if using `ingest/genomes/genomes_pipeline.py`
 
 # Install
+
 ### Docker
+
 If on Apple silicon Mac (e.g. M1), do:
```

Comment:
```suggestion
### Docker

If <some other reason>, do:
```

Following [comment from previous PR](https://github.com/broadinstitute/scp-ingest-pipeline/pull/293#pullrequestreview-1301606674), let's rearrange these subsections so "Native" is on top, and "Docker" below it, given that native local development is again generally feasible.  

Developing natively (though still via virtualenv) seems faster and simpler than developing in Docker, and thus typically preferable, though the latter might make more sense in special cases.  Re-ordering the sections would reflect that.

Similarly, "If on Apple silicon Mac (e.g. M1)," rationale also seems worth updating -- if my above understanding is correct.

Reply:
Updated
`0842d7d`